### DISCUSSION TITLE
在WPF中使用winSDK

### DISCUSSION THREAD

**Discussion by weqasds** [BYJRK/DotNet-Discussions#107] [通用]
我希望在WPF中使用winui3，在winui3的路线图中，winui3的控件已经可以通过xaml island在wpf中使用 而winui3集成在winSDK中，所以我该如何在wpf使用winSDK

**Comment by BYJRK** [comment]
你具体想要使用Win UI 3里面的什么控件？一般情况下，使用一个第三方控件库（比如WPFUI，ModernWPFUI，HandyControl），或者WebView2之类的就已经能满足大多数要求了

**Reply by weqasds** [reply]
主要是导航栏
